### PR TITLE
Strings reworked 7 redux

### PR DESCRIPTION
**Pull request by comradekingu**
Seems good to me.

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -13,72 +13,72 @@
...
+    <string name="select_color">Select colour</string>
     <string name="rendering_value_thick_name">Thick</string>
-    <string name="desert_render_descr">For deserts and other sparsely populated areas. Displays more detail on a viewing scale.</string>
+    <string name="desert_render_descr">For deserts and other sparsely populated areas. More details.</string>
     <string name="shared_string_downloading_formatted">Downloading %s</string>
-    <string name="edit_profiles_descr">You can’t delete default OsmAnd profiles, but you can disable them in previous screen, or move them to the bottom.</string>
+    <string name="edit_profiles_descr">You can’t delete default OsmAnd profiles, but you can go back and remove them, or move them to the bottom.</string>
     <string name="edit_profiles">Edit profiles</string>
-    <string name="select_nav_profile_dialog_message">Navigation type affects the rules for route calculations.</string>
+    <string name="select_nav_profile_dialog_message">\'Navigation type\' affects rules for route calculations.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]: Not completely sure about this "navigation type" markup

File: OsmAnd/res/values/strings.xml [hunk 2]
```diff
@@ -270,6 +270,7 @@
     <string name="default_speed_setting_title">Default speed</string>
     <string name="default_speed_setting_descr">Change default speed settings</string>
     <string name="minmax_speed_dialog_title">Set min/max speed</string>
+    <string name="default_speed_dialog_msg">Used to calculate the expected arrival time</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
```

File: OsmAnd/res/values/strings.xml [hunk 3]
```diff
@@ -13,72 +13,72 @@
...
+    <string name="added_profiles_descr">Profiles added by plugin</string>
     <string name="shared_string_turn_off">Turn off</string>
     <string name="new_plugin_added">New plugin added</string>
     <string name="clear_confirmation_msg">Clear %1$s?</string>
     <string name="shared_string_revert">Revert</string>
     <string name="track_saved">Track saved</string>
-    <string name="empty_filename">File name is empty</string>
-    <string name="default_speed_dialog_msg">Used to estimate arrival time for unknown type of roads and to limit speed for all roads (could change the route)</string>
+    <string name="empty_filename">Filename empty</string>
+    <string name="default_speed_dialog_msg">Estimates arrival time for unknown road types, and to limit speed for all roads (can re-oute)</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="default_speed_dialog_msg">Estimates arrival time for unknown road types, and limits speed for all roads (can re-oute)</string>
```

File: OsmAnd/res/values/strings.xml [hunk 4]
```diff
@@ -1093,6 +1091,7 @@
     <string name="osm_live_subscription_desc">Subscription enables hourly, daily, weekly updates, and unlimited downloads for all maps globally.</string>
     <string name="get_it">Get it</string>
     <string name="get_for">Get for %1$s</string>
+    <string name="get_for_month">Get for %1$s month</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
```

File: OsmAnd/res/values/strings.xml [hunk 5]
```diff
@@ -3478,7 +3477,7 @@
     <string name="routing_attr_difficulty_preference_name">Prefered difficulty</string>
     <string name="routing_attr_difficulty_preference_description">Prefer routes of this difficulty, although routing over harder or easier pistes is still possible if shorter.</string>
     <string name="routing_attr_freeride_policy_name">Off-piste</string>
-    <string name="routing_attr_freeride_policy_description">Freeride and offpiste are unofficial routes and passages. Typically ungroomed, unmaintained by the officials and not checked in the evening. Enter at your own risk.</string>
+    <string name="routing_attr_freeride_policy_description">\'Freeride\' and \'Off-piste\' are unofficial routes and passages. Typically ungroomed, unmainted by the officials and not checked in the evening. Enter at your own risk.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="routing_attr_freeride_policy_description">\'Freeride\' and \'Off-piste\' are unofficial routes and passages. Typically ungroomed, unmaintained by the officials and not checked in the evening. Enter at your own risk.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 6]
```diff
@@ -3374,8 +3373,8 @@
     <string name="favorite_empty_place_name">Place</string>
     <string name="quick_action_duplicates">Quick action renamed to %1$s to avoid duplication.</string>
     <string name="quick_action_duplicate">Quick action name duplicate</string>
-    <string name="quick_action_showhide_favorites_descr">A toggle to show or hide the Favorite points on the map.</string>
-    <string name="quick_action_showhide_poi_descr">A toggle to show or hide POIs on the map.</string>
+    <string name="quick_action_showhide_favorites_descr">Button showing or hideing Favorite points on the map.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="quick_action_showhide_favorites_descr">Button showing or hiding \'Favorite\' points on the map.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 7]
```diff
@@ -3331,7 +3330,7 @@
     <string name="number_of_contributors">Number of contributors</string>
     <string name="number_of_edits">Number of edits</string>
     <string name="reports_for">Report for</string>
-    <string name="file_name_containes_illegal_char">File name contains illegal character</string>
+    <string name="file_name_containes_illegal_char">Filename contains illegal character</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="file_name_containes_illegal_char">Remove all illegal characters from the filename</string>
```

File: OsmAnd/res/values/strings.xml [hunk 8]
```diff
@@ -3077,11 +3076,11 @@
     <string name="search_offline_address">Offline search</string>
     <string name="search_online_address">Online search</string>
     <string name="max_level_download_tile">Max. online zoom</string>
-    <string name="max_level_download_tile_descr">Do not browse online map tiles for zoom levels beyond this.</string>
+    <string name="max_level_download_tile_descr">Do not browse online maps for zoom levels beyond this.</string>
     <string name="route_general_information">Total distance %1$s, traveling time %2$d h %3$d min.</string>
     <string name="router_service_descr">Online or offline navigation service.</string>
     <string name="router_service">Navigation service</string>
-    <string name="sd_dir_not_accessible">The data storage folder on the memory card is not accessible!</string>
+    <string name="sd_dir_not_accessible">The data folder on the memory card is not accessible!</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="sd_dir_not_accessible">The storage folder on the memory card is inaccessible!</string>
```

File: OsmAnd/res/values/strings.xml [hunk 9]
```diff
@@ -1236,20 +1233,20 @@
...
     <string name="upload_anonymously">Upload anonymously</string>
     <string name="show_transparency_seekbar">Show transparency seekbar</string>
-    <string name="download_files_error_not_enough_space">Not enough space!
-        {3} MB is needed temporarily, {1} MB permanently.
-        (Only {2} MB available.)</string>
-    <string name="download_files_question_space_with_temp">Download {0} file(s)?
-        {3} MB is used temporarily, {1} MB permanently. (Of {2} MB.)</string>
-    <string name="download_files_question_space">Download {0} file(s)?
-        {1} MB (of {2} MB) will be used.</string>
+    <string name="download_files_error_not_enough_space">Not enough space!\n
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]: Can't remember why I put in lineshifts. Possibly to do with the Weblate editor.

File: OsmAnd/res/values/strings.xml [hunk 10]
```diff
@@ -2500,6 +2497,7 @@
     <string name="map_widget_transparent">Transparent widgets</string>
     <string name="bg_service_sleep_mode_off">Run\n app in background</string>
     <string name="bg_service_sleep_mode_on">Stop\n running in background</string>
+    <string name="gps_wakeup_interval">GPS wake-up interval: %s</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
```

File: OsmAnd/res/values/strings.xml [hunk 11]
```diff
@@ -3331,7 +3329,7 @@
     <string name="number_of_contributors">Number of contributors</string>
     <string name="number_of_edits">Number of edits</string>
     <string name="reports_for">Report for</string>
-    <string name="file_name_containes_illegal_char">File name contains illegal character</string>
+    <string name="file_name_containes_illegal_char">Remove all illegal characters from the filename</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="file_name_containes_illegal_char">Remove all illegal characters from the filename first</string>
```

File: OsmAnd/res/values/strings.xml [hunk 12]
```diff
@@ -2612,14 +2609,14 @@
     <string name="left_side_navigation">Left-hand traffic</string>
     <string name="left_side_navigation_descr">For countries where people drive on the left side of the road.</string>
     <string name="local_index_description">Tap any existing item to see more details, long-tap to deactivate or delete. Current data on device (%1$s free):</string>
-    <string name="unknown_from_location">Point of departure not yet determined.</string>
-    <string name="unknown_location">Position not yet known.</string>
+    <string name="unknown_from_location">Point of departure not yet determined</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="unknown_from_location">Point of departure not yet determined.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 13]
```diff
@@ -2613,13 +2610,13 @@
     <string name="left_side_navigation_descr">For countries where people drive on the left side of the road.</string>
     <string name="local_index_description">Tap any existing item to see more details, long-tap to deactivate or delete. Current data on device (%1$s free):</string>
     <string name="unknown_from_location">Point of departure not yet determined.</string>
-    <string name="unknown_location">Position not yet known.</string>
+    <string name="unknown_location">Position not yet known</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="unknown_location">Position not yet known.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 14]
```diff
@@ -3503,7 +3501,7 @@
     <string name="routing_attr_difficulty_preference_name">Prefered difficulty</string>
     <string name="routing_attr_difficulty_preference_description">Prefer routes of this difficulty, although routing over harder or easier pistes is still possible if shorter.</string>
     <string name="routing_attr_freeride_policy_name">Off-piste</string>
-    <string name="routing_attr_freeride_policy_description">Freeride and offpiste are unofficial routes and passages. Typically ungroomed, unmaintained by the officials and not checked in the evening. Enter at your own risk.</string>
+    <string name="routing_attr_freeride_policy_description">\'Freeride\' and \'Off-piste\' are unofficial routes and passages. Typically ungroomed, unmaintained by the officials and not checked in the evening. Enter at your own risk.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="routing_attr_freeride_policy_description">\'Freeride\' and \'Off-piste\' are unofficial routes and passages. Typically ungroomed, unmaintained and not checked in the evening. Enter at your own risk.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 15]
```diff
@@ -3503,7 +3501,7 @@
     <string name="routing_attr_difficulty_preference_name">Prefered difficulty</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="routing_attr_difficulty_preference_name">Preferred difficulty</string>
```

File: OsmAnd/res/values/strings.xml [hunk 16]
```diff
@@ -3418,7 +3416,7 @@
     <string name="quick_action_gpx_category_descr">Choose an optional category.</string>
     <string name="quick_action_poi_list">POI list</string>
     <string name="quick_action_sh_poi_descr">Add one or more POI categories to display on the map.</string>
-    <string name="quick_action_page_list_descr">A button to page through the list below.</string>
+    <string name="quick_action_page_list_descr">Button paging through the list below.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]: Alternatively "flipping through"

File: OsmAnd/res/values/strings.xml [hunk 17]
```diff
@@ -3381,16 +3379,16 @@
...
+    <string name="quick_action_add_marker_descr">Button adding a map marker in the middle of the screen.</string>
+    <string name="quick_action_add_gpx_descr">Button adding a GPX waypoint in the middle of the screen.</string>
+    <string name="quick_action_take_audio_note_descr">Button adding an audio note in the middle of the screen.</string>
+    <string name="quick_action_take_video_note_descr">Button adding a video note in the middle of the screen.</string>
+    <string name="quick_action_take_photo_note_descr">Button adding a photo note in the middle of the screen.</string>
+    <string name="quick_action_add_osm_bug_descr">Button adding an OSM note in the middle of the screen.</string>
+    <string name="quick_action_add_poi_descr">Button adding a POI in the middle of the screen.</string>
+    <string name="quick_action_navigation_voice_descr">Button turning on or off voice guidance during navigation.</string>
+    <string name="quick_action_add_parking_descr">Button adding a parking location in the middle of the screen.</string>
+    <string name="quick_action_switch_day_night_descr">Button switching between \'Day\' and \'Night\' modes for OsmAnd.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="quick_action_switch_day_night_descr">Button switching between \'Day\' and \'Night\' mode.</string>
```

Comment (comradekingu) [reply OsmAnd/res/values/strings.xml]: Spelling: - for OsmAnd

File: OsmAnd/res/values/strings.xml [hunk 18]
```diff
@@ -3189,7 +3187,7 @@
     <string name="app_mode_pedestrian">Walking</string>
     <string name="position_on_map_center">Center</string>
     <string name="position_on_map_bottom">Bottom</string>
-    <string name="navigate_point_top_text">Input latitude &amp; longitude in the selected format (D - degrees, M - minutes, S - seconds)</string>
+    <string name="navigate_point_top_text">Input latitude and longitude in the selected format (D - degrees, M - minutes, S - seconds)</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="navigate_point_top_text">Input latitude and longitude as degrees-minutes-seconds (DMS)</string>
```
https://en.wikipedia.org/wiki/Degree_(angle)#Subdivisions

File: OsmAnd/res/values/strings.xml [hunk 19]
```diff
@@ -11,103 +11,103 @@
...
     <string name="login_and_pass">Username and password</string>
-    <string name="plugin_global_prefs_info">This plugin settings are global, and apply to all profiles.</string>
-    <string name="osm_editing">OpenStreetMap Editing</string>
-    <string name="osm_edits_view_descr">You can view all your unuploaded edits or osm bugs in Menu — My places — OSM Edits. Uploaded points don’t show in OsmAnd.</string>
+    <string name="plugin_global_prefs_info">These settings apply to all profiles.</string>
+    <string name="osm_editing">OSM editing</string>
+    <string name="osm_edits_view_descr">View all your unuploaded edits or OSM bugs in \'Menu\' → \'My places\' — \'OSM edits\'. Uploaded points don\'t show up in OsmAnd.</string>
     <string name="app_mode_osm">OSM</string>
-    <string name="select_nav_icon_descr">You will see the icon only while navigation or while moving.</string>
-    <string name="select_map_icon_descr">Map icon appears only on the map, and changing while navigation to navigation icon.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]: Not really sure what the second part of this is detailing…

File: OsmAnd/res/values/strings.xml [hunk 20]
```diff
@@ -11,103 +11,103 @@
...
     <string name="clear_confirmation_msg">Clear %1$s?</string>
     <string name="shared_string_revert">Revert</string>
     <string name="track_saved">Track saved</string>
-    <string name="empty_filename">File name is empty</string>
-    <string name="default_speed_dialog_msg">Used to estimate arrival time for unknown type of roads and to limit speed for all roads (could change the route)</string>
+    <string name="empty_filename">Filename empty</string>
+    <string name="default_speed_dialog_msg">Estimates arrival time for unknown road types, and limits speed for all roads (can re-oute)</string>
     <string name="rendering_value_white_name">White</string>
-    <string name="swap_two_places">Swap %1$s and %2$s</string>
+    <string name="swap_two_places">%1$s ⇄ %2$s</string>
```
Comment (vshcherb) [inline OsmAnd/res/values/strings.xml]: This probably won't be obvious in UI so better to keep the word

Comment (comradekingu) [inline OsmAnd/res/values/strings.xml]:
```suggestion
    <string name="swap_two_places">Swap %1$s and %2$s</string>
```

**Pull request merged**